app/controllers/hub/efile_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Hub
+  class EfileController < Hub::BaseController
+    include FilesConcern
+    authorize_resource
```

Comment:
we should always load the resource first and then authorize it because otherwise the authorize will have nothing to check 

see more here:https://docs.google.com/document/d/10WNXTBOHogniT1YWcNR2q7EdGmcDOKznAQXRQZnoLHQ/edit or here https://github.com/codeforamerica/vita-min/pull/3481/files

Reply:
I don't have access to the Google docs, can you grant me access @embarnard?